fluent-package/yum/systemd-test/update-to-next-version.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+set -exu
+
+. $(dirname $0)/commonvar.sh
+
+# Install the current
+package="/vagrant/${distribution}/${DISTRIBUTION_VERSION}/x86_64/Packages/fluent-package-[0-9]*.rpm"
+sudo $DNF install -y $package
+sudo systemctl enable --now fluentd
+systemctl status --no-pager fluentd
+
+# Make a dummy pacakge for the next version
+case $distribution in
+    amazon)
+        sudo amazon-linux-extras install -y epel
+        ;;
+    *)
+        sudo $DNF install -y epel-release
+        ;;
+esac
+sudo $DNF install -y rpmrebuild
+# Example: "1.el9"
+release=$(rpmquery --queryformat="%{Release}" -p $package)
+# Example: "1"
+release_ver=$(echo $release | sed -E "s/([0-9]+)\.(.+)/\1/g")
+# Example: "2.el9"
+next_release=$(echo $release | sed -E "s/([0-9]+)\.(.+)/$(($release_ver+1)).\2/g")
+rpmrebuild --release=$next_release --package $package
```

Comment:
How about this?

next_release=$(($release_ver+1)).$(echo $release | cut -d. -f2)

Reply:
Cool! Thanks! Fixed!